### DISCUSSION TITLE
Latest upgrade busted my autoshutdown! (Solved)

### DISCUSSION THREAD

**Discussion by librarymark** [jamesmccannon02/OctoPrint-Tplinkautoshutdown#52] [General]
I updated the plugin and now it does not work, it is not able to show the device config. I have my plug IP static. What changed today?

**Comment by CodingTheUnknown** [comment]
Hi, did you read the release notes? You need to re-select the device type. click save and restart the server.

**Comment by librarymark** [comment]
Thanks! That did it.
